schema_salad/dlang_codegen.py

```diff
@@ -70,6 +71,28 @@ def epilogue(self, root_loader: TypeDef) -> None:
 ///
 alias importFromURI = import_!DocumentRootType;
 """)
+        if self.examples:
```

Comment:
Thank you for the information!

Are there equivalent examples for CWL v1.0 and v1.1?

Reply:
Added!

I found that some files cannot be loaded due to schema-salad-d does not consider the case of unions with `Any`.
It is a matter of parser by schema-salad-d rather than the generated code by schema-salad-tool and therefore I can fix it without re-generating the parser :-)

Reply:
You are welcome!

For CWL v1.0 and v1.1, we haven't made them yet.
You could try the following from a checkout of https://github.com/common-workflow-language/cwl-v1.1/tree/main/tests and https://github.com/common-workflow-language/common-workflow-language/tree/main/v1.0/examples
```
git ls-files -- tests | grep cwl$ | xargs -I{} bash -c 'cwltool --validate {} >/dev/null 2>/dev/null && echo {}' > valid_examples.txt
mkdir valid-examples; for file in $(cat valid_examples.txt); do cp ${file} valid-examples/valid_$(basename ${file}) ; done
```

Reply:
I added tests for CWL v1.0 and v1.1 and confirmed tests for v1.0 is passed!

I also fix the case of unions with `Any` but I found another issue with `hints` that has extended process requirements and uses map notation.
To make them passed, extended fields must be supported by schema-salad-d.

Reply:
> I found another issue with hints that has extended process requirements and uses map notation.

Solved!

Reply:
I mark it resolved because tests are already added!